### PR TITLE
fix: slow down our internal job to deal with the TMDB 429 :(

### PR DESCRIPTION
**Pull request by AndreasPB**
# Description

We are too #blazinglyfast

Will hopefully fix our throttling problem

### CODE REVIEW

File: backend/cli.py [hunk 1]
```diff
@@ -228,6 +228,8 @@ def update_media(chunk_size: int, popularity: float = 1):
                 )
                 if res.status_code != 200:
                     echo_warning(res.text)
+                else:
+                    print(res.text)
```
Comment (AndreasPB) [inline backend/cli.py:232]: For debugging - Should be removed or done in a smarter way (the sum of added media or smth)

File: internal/routes.go [hunk 2]
```diff
@@ -140,6 +142,16 @@ func fetchMovie(id string, movieCh chan Movie) {
 
 	defer res.Body.Close()
 
+	if res.StatusCode == 429 {
```
Comment (nullxDEADBEEF) [inline internal/routes.go:145]: can we make use of the http built-in constants? e.g. `http.StatusOK`

File: internal/routes.go [hunk 3]
```diff
@@ -140,6 +142,16 @@ func fetchMovie(id string, movieCh chan Movie) {
...
+		fmt.Println(res.Status, id, "- Will retry in 1 second")
+		time.Sleep(1)
+		fetchMovie(id, movieCh)
+	}
+
+	if res.StatusCode != 200 {
```
Comment (nullxDEADBEEF) [inline internal/routes.go:151]: same as above

**Review by nullxDEADBEEF — CHANGES_REQUESTED** [review]
see comments

**Review by Primdahl26 — APPROVED** [review]

File: internal/routes.go (hunk 2 above)
Comment (AndreasPB) [reply internal/routes.go:145]: Ah, good catch! I'm changing 429 to `http.StatusTooManyRequests` as well

File: internal/routes.go (hunk 3 above)
Comment (AndreasPB) [reply internal/routes.go]: Done - Did both for TV and Person as well

File: internal/routes.go (hunk 2 above)
Comment (nullxDEADBEEF) [reply internal/routes.go]: Awesome 🙌

File: internal/routes.go (hunk 3 above)
Comment (nullxDEADBEEF) [reply internal/routes.go]: Beautiful 🤘

**Pull request merged**